### ISSUE TITLE
SetOption73 is not "Enable HTTP CORS" in 8.3.1(tasmota)

### ISSUE THREAD

**Issue by Ryan-J** [IOTReboot/TasUI#18]
As noted below, SetOption73 1 decouples buttons and the corresponding relays will not be triggered:
https://github.com/arendst/Tasmota/pull/8177

Doing a device discovery disabled all my light switches.

**Comment by madivad** [comment]
A HUGE bug, I'd just come here to report that. 
a) it should not be enabled by default (it will break devices), ie, this "feature" should be defaulted to not use CORS (and have a disclaimer about it),
b) there should be a warning it will break newer (8.3.1) devices

**Comment by Ryan-J** [comment]
It doesn't permanently break them. Issuing SetOption73 0 will get them back to normal (unless you are actually using SetOption73 1, which I do in some cases).

**Comment by foxsam21** [comment]
Yeah. I just used TasUI for the first time and all my buttons stopped working. Lucky that i thought of checking SetOption73 so I figured it out quickly and set all my devices back to `so73 0`. This should be changed asap.
